source/_integrations/shelly.markdown

```diff
@@ -43,34 +43,40 @@ Integrate [Shelly devices](https://shelly.cloud) into Home Assistant.
+4. In **CoIoT peer** field, enter `Your_Home_Assistant_local_IP_address:5683`
+5. Click **Save**;
+6. Reboot the Shelly device.
 
-- Check if your Shelly devices have a properly configured `CoIoT peer`.
-- If you can't find the `CoIoT peer` settings in the device's web panel, it's probably using an ancient firmware version, and you should update it to the current one.
-- If Shelly devices are in a different subnet than the Home Assistant server, you should ensure communication on `UDP` port `5683` between these subnets.
-- If Home Assistant is running as a virtual machine or service on an operating system other than Home Assistant OS, you should open `UDP` port `5683` on the device's firewall and/or ensure that communication from this port is redirected to the Home Assistant service.
-- The missing push updates may be related to the WiFi network range. If using a WiFi network with several access points, enable **Internet & Security** >> **WiFi Client AP Roaming** option. Consider moving Shelly device closer to the WiFi access point. Consider adding another WiFi access point, which will improve the connection quality with the device.
-- If you think your Shelly devices are working correctly and don't want to change your network/configuration, you can ignore the repair issue. Still, you must know you are giving up the best experience of using first-generation Shelly devices with Home Assistant.
+If you have followed the above steps, but the repair issue remains, there are diagnostic steps and ideas listed in the **Known issues and limitations** section at the bottom of this document.
 
-## Shelly device configuration (generation 2)
 
-Generation 2 devices use the `RPC` protocol to communicate with the integration. Battery powered devices need manual outbound WebSocket configuration, Navigate to the local IP address of your Shelly device, **Settings** >> **Connectivity** >> **Outbound WebSocket** and check the box **Enable Outbound WebSocket**, under server enter the following address:
+## Configuring 2nd generation Shelly devices
 
-`ws://` + `Home_Assistant_local_ip_address:Port` + `/api/shelly/ws` (for example: `ws://192.168.1.100:8123/api/shelly/ws`), click **Apply** to save the settings.
+Second generation Shelly devices use the `RPC` protocol to communicate with the integration. Battery powered 2nd generation devices **only** need manual outbound WebSocket configuration:
+
+1. Ensure the Shelly device is on the current firmware. Older firmware will lack the necessary setting;
+2. In the device's web UI (at the local IP address of your Shelly device), go to: Left sidebar >> **Settings** >> **Outbound WebSocket**;
+3. Make sure box **Enable Outbound WebSocket** is selected;
+4. In Server field, enter `ws://Your_Home_Assistant_local_IP_address:Port/api/shelly/ws` (note: port must match your Home Assistant port when you open the UI);
+5. You *might* have to tweak the **Connection Type**, depending on your local TLS certificate situation;
```

Comment:
I would add the most common one and then say "you may need to tweak...."

Reply:
I wouldn't say there's a common one - different folks run their networks differently. If somebody isn't deliberate with their network security there are only 3 options and they can try "whichever one works" but I'm loathe to put it in writing like that.